db/blob/blob_source.cc

```diff
@@ -152,6 +153,140 @@ Status BlobSource::GetBlob(const ReadOptions& read_options,
   return s;
 }
 
+void BlobSource::MultiGetBlob(
```

Comment:
I'm wondering if it would make sense to push more of the logic into `BlobSource`, like the logic that sorts blobs by offset for example.

Reply:
That makes sense to me. Since I also need to modify 'VersionSet`'s MultiGetBlob, we can put them in a new PR.

Reply:
Added: https://github.com/facebook/rocksdb/issues/10156#issuecomment-1163469932